frontend/maithili-language-explorer/style.css

```
@@ -0,0 +1,582 @@
+  --maithili-card-hover: rgba(255, 255, 255, 1);
+  --maithili-text: #1a202c;
+  --maithili-text-muted: #4a5568;
+  --maithili-alt-bg: #f5ede4;
+}
+
+.maithili-main {
+  min-height: 100vh;
+  padding-top: 72px;
+  background-color: var(--maithili-dark-bg);
+  color: var(--maithili-text);
+  font-family: 'Outfit', sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Outfit" (font-family-name-quotes)

(font-family-name-quotes)